FairBid/README.mb

```diff
@@ -0,0 +1,3 @@
+*SKAdnetwork PLIST for FairBid*
+
+This directory containes that plist entries for each network integrated with FairBid mediation platform. This is the basis for the plist generator in [FairBid Documentation](https://developer.fyber.com/hc/en-us/articles/360010077777-Supported-Networks#platform-ios)
```

Comment:
This directory containes ~that~ **the** plist entries for each network integrated with FairBid mediation platform **on iOS**. This is the basis for the plist generator in **the** [FairBid Documentation](https://developer.fyber.com/hc/en-us/articles/360010077777-Supported-Networks#platform-ios)